WordPressAuthenticator.podspec

```diff
@@ -1,20 +1,23 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressAuthenticator"
   s.version       = "1.36.0-beta.2"
-  s.summary       = "WordPressAuthenticator implements an easy and elegant way to authenticate your WordPress Apps."
 
+  s.summary       = "WordPressAuthenticator implements an easy and elegant way to authenticate your WordPress Apps."
   s.description   = <<-DESC
                     This framework encapsulates everything required to display the Authentication UI
                     and perform authentication against WordPress.com and WordPress.org sites.
 
-                    Plus: WordPress.com *signup*  is supported.
+                    Plus: WordPress.com *signup* is supported.
                     DESC
 
-  s.homepage      = "https://github.com/wordpress-mobile/WordPressAuthenticator-iOS"
-  s.license       = "GPLv2"
-  s.author        = { "WordPress" => "mobile@automattic.com" }
+  s.homepage      = "http://apps.wordpress.com"
+  s.license       = { :type => "GPLv2", :file => "LICENSE" }
```

Comment:
* We happen to have a `LICENSE` file in that repo, so better point to it.
* But that's apparently not the case for all our repos, so I wonder what's best for other pods for which I'll open similar PRs: create the `LICENSE` file and point to it, or just not list the file name if the file doesn't exist, without creating it?
* It's easy enough to copy/paste the `LICENSE` file we have here to any pod that also uses GPLv2 but doesn't have a `LICENSE` file yet. But not all our pods use `GPLv2` (some use `MIT` license). Not sure that license discrepency is expected (but that's probably a debate for another day)

Reply:
The only possible issue I see with this format is if we chose to change license, by means of updating the `LICENSE` file, and forgot to update the `type` key here.

IMHO, it'd be great to have a `LICENSE` file in every open-source repo. I'm not sure if it's an explicit standard, but I've come to expect it from the sample of open-source repos I've interact with so far.

Reply:
The LICENSE file is indeed a standard in OSS world.

Reply:
Yeah, the `MIT` dependencies are tricky – for now I'd say we just put the MIT License in the `LICENSE` file and we'll figure out aligning licenses another day